.github/workflows/release-e2e.yaml

```diff
@@ -0,0 +1,51 @@
+name: release-e2e
+
+on:
+  release:
+    types: [created]
+  workflow_dispatch:
+
+jobs:
+  release-e2e-kind:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v1
+    - uses: actions/setup-go@v2
+      with:
+        go-version: '~1.18'
+
+    - name: install_kind_binary
+      env:
+        KIND_VERSION: v0.12.0
+        KIND_OS: kind-linux-amd64
+      run: |
+        curl -sLo kind "https://github.com/kubernetes-sigs/kind/releases/download/${KIND_VERSION}/${KIND_OS}"
+        chmod +x kind
+        sudo mv kind /bin/
+
+    - name: start_kind_cluster
+      run: |
+        # /bin/kind create cluster
+        make kind-cluster
+
+
+    - name: install_kubectl
+      run: |
+        curl -LO "https://dl.k8s.io/release/$(curl -L -s https://dl.k8s.io/release/stable.txt)/bin/linux/amd64/kubectl"
+        sudo install -o root -g root -m 0755 kubectl /usr/local/bin/kubectl
+
+    - name: release-install
+      needs: [start_kind_cluster, install_kubectl]
+      run: |
+        kubectl apply -f https://github.com/cert-manager/cert-manager/releases/download/v1.7.1/cert-manager.yaml
+        kubectl wait --for=condition=Available --namespace=cert-manager deployment/cert-manager-webhook --timeout=60s
+        kubectl apply -f https://github.com/operator-framework/rukpak/releases/download/v0.3.0/rukpak.yaml
```

Comment:
Should we wait for the plain provisioner to rollout as well, like we do for cert-manager? Something like
```
kubectl wait --for=condition=Available --namespace=rukpak-system deployment/plain-provisioner --timeout=60s
```

Reply:
That makes sense to me. Will add.

Reply:
Also the webhook. I've noticed local e2e runs that fail because the webhook pod isn't serving by the time we start hammering the API server with bundles.

Reply:
@joelanford I've seen this exact behavior locally when copy-pasting our installation release instructions. Do we need an issue for fixing this behavior?

Reply:
Sorry, beginner question - a rollout wait for another webhook beyond the cert manager one (i.e. not just this `kubectl wait --for=condition=Available --namespace=cert-manager deployment/cert-manager-webhook --timeout=60s`?)

Reply:
Yes, I would add some checks to ensure that the additional resources and webhooks are up and available before starting the e2e tests. Something similar to #328 
```
	kubectl wait --for=condition=Available --namespace=rukpak-system deployment/plain-provisioner --timeout=60s
	kubectl wait --for=condition=Available --namespace=rukpak-system deployment/rukpak-core-webhook --timeout=60s
	kubectl wait --for=condition=Available --namespace=crdvalidator-system deployment/crd-validation-webhook --timeout=60s
```

Reply:
Pulled checks from the new release instructions.